client/coral-admin/src/components/CommentList.css

```diff
@@ -36,13 +36,33 @@
 
 .listItem {
   border-bottom: 1px solid #e0e0e0;
-  padding: 16px;
   font-size: 16px;
+  width: 100%;
+  max-width: 660px;
+  min-width: 400px;
+  margin: 0 auto;
+  padding: 16px 14px;
+  position: relative;
+  transition: box-shadow 200ms;
+
+
+  &:hover {
```

Comment:
Let me dive deeper about this concept before renaming all the files. We can do that in another PR.